polaris/hub/polarisfs.py

```diff
@@ -105,6 +124,14 @@ def ls(
             for p in response.json()
         ]
 
+        if self.use_listings_cache:
+            self.dircache[path] = detailed_listings
+
+        if not detail:
+            return [p["name"].removeprefix(self.prefix) for p in response.json()]
```

Comment:
`removeprefix` is not needed anymore right? Already done when creating `detailed_listings`?